packages/utils/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@ponder/utils",
+  "version": "0.0.0",
```

Comment:
We should think about how we want to version this package. My instinct is that we should lock it with the other packages, but could be convinced otherwise.